src/main/scala/com/codacy/analysis/cli/utils/EitherOps.scala

```diff
@@ -0,0 +1,34 @@
+package com.codacy.analysis.cli.utils
+
+import scala.util.{Failure, Success, Try}
+
+object EitherOps {
+
+  def sequenceWithFixedLeftUnit[A](left: A)(eitherIterable: Seq[Either[A, Unit]]): Either[A, Unit] = {
```

Comment:
"sequenceUnitWithFixedLeft" makes more sense to me, if you want to add "Unit" to the name.